packages/sit-onyx/src/styles/variables-onyx.css

```diff
@@ -0,0 +1,164 @@
+  --onyx-color-universal-orange-1000: #73451c;
+  --onyx-color-universal-orange-1100: #593412;
+  --onyx-color-universal-orange-1200: #402309;
+  --onyx-color-universal-green-100: #ecf8f2;
+  --onyx-color-universal-green-200: #c2e4d3;
+  --onyx-color-universal-green-300: #98d1b3;
+  --onyx-color-universal-green-400: #6ebe94;
+  --onyx-color-universal-green-500: #44aa75;
+  --onyx-color-universal-green-600: #3b9b69;
+  --onyx-color-universal-green-700: #328c5e;
+  --onyx-color-universal-green-800: #297d52;
+  --onyx-color-universal-green-900: #216d46;
+  --onyx-color-universal-green-1000: #185e3a;
+  --onyx-color-universal-green-1100: #064023;
+  --onyx-color-universal-green-1200: #064023;
+  --onyx-color-universal-purple-100: #f4f1f8;
+  --onyx-color-universal-purple-200: #e6dcf2;
+  --onyx-color-universal-purple-300: #cab4e5;
+  --onyx-color-universal-purple-400: #aa89d1;
+  --onyx-color-universal-purple-500: #9068c0;
+  --onyx-color-universal-purple-600: #7b52ab;
+  --onyx-color-universal-purple-700: #613990;
+  --onyx-color-universal-purple-800: #3a1664;
+  --onyx-number-spacing-050: 0;
+  --onyx-color-universal-purple-900: #210640;
+  --onyx-color-universal-purple-1000: #210640;
+  --onyx-number-spacing-100: 0.125rem;
+  --onyx-color-universal-purple-1100: #210640;
+  --onyx-color-universal-purple-1200: #210640;
+  --onyx-number-spacing-200: 0.25rem;
+  --onyx-number-spacing-300: 0.5rem;
+  --onyx-number-spacing-400: 1rem;
+  --onyx-number-spacing-500: 1.5rem;
+  --onyx-number-spacing-600: 2rem;
+  --onyx-number-spacing-700: 3rem;
+  --onyx-number-spacing-800: 4rem;
+  --onyx-number-spacing-900: 6rem;
+  --onyx-number-size-050: 0.0625rem;
+  --onyx-number-size-100: 0.125rem;
+  --onyx-number-size-150: 0.25rem;
+  --onyx-number-size-200: 0.5rem;
+  --onyx-number-size-250: 0.625rem;
+  --onyx-number-size-300: 0.75rem;
+  --onyx-number-size-350: 1rem;
+  --onyx-number-size-400: 1.25rem;
+  --onyx-number-size-450: 1.5rem;
+  --onyx-number-size-500: 2rem;
+  --onyx-number-size-550: 2.5rem;
+  --onyx-number-size-600: 3rem;
+  --onyx-number-size-650: 3.375rem;
+  --onyx-number-size-700: 3.625rem;
+  --onyx-number-size-750: 4rem;
+  --onyx-number-size-800: 4.5rem;
+  --onyx-number-size-850: 5rem;
+  --onyx-number-size-900: 6rem;
+  --onyx-number-size-950: 8rem;
+  --onyx-number-radius-100: 0;
+  --onyx-number-radius-200: 0.125rem;
+  --onyx-number-radius-300: 0.25rem;
+  --onyx-number-radius-400: 0.5rem;
+  --onyx-number-radius-500: 1rem;
+  --onyx-number-radius-600: 2rem;
+  --onyx-number-radius-700: 62.5rem;
+  --onyx-number-opacity-100: 6.25rem;
+  --onyx-number-opacity-200: 5.625rem;
+  --onyx-number-opacity-300: 5rem;
+  --onyx-number-opacity-400: 4.375rem;
+  --onyx-number-opacity-500: 3.75rem;
+  --onyx-number-opacity-600: 3.125rem;
+  --onyx-number-opacity-700: 2.5rem;
+  --onyx-number-opacity-800: 1.875rem;
+  --onyx-number-opacity-900: 1.25rem;
+  --onyx-number-opacity-950: 0.625rem;
+  --onyx-number-opacity-1000: 0;
```

Comment:
@JoCa96 @BoppLi We convert all numbers to rem but for opacity this is not correct, right?

Reply:
opacity should be a unitless number I think

Reply:
Then the question is how we can generically detect this in the CLI...should we check if the variable name contains "opacity"?

Reply:
If we don't need opacity tokens, we should remove them :)

Reply:
remove opacity tokens or stick with them. I don't care because from ux-perspective I cannot work with them in figma.

Reply:
Lets remove them then